docs/deprecations.md

```diff
@@ -20,6 +20,12 @@ Please upgrade to use at least Node.js v10 (It's recommended to use LTS version,
 
 Please use `onUnauthenticatedRequest` instead. `allowUnauthenticated` will be removed with v2.0.0
 
+<a name="AWS_HTTP_API_VERSION"><div>&nbsp;</div></a>
+
+## AWS HTTP API payload format
+
+Default HTTP API Payload version will be switched to 2.0 with next major release. Please configure `payload` explicitly.
```

Comment:
Here it's desired to be more descriptive, therefore let's maybe change it to:

--- 

Default HTTP API Payload version will be switched to 2.0 with next major release (for more details see [payload format documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html#http-api-develop-integrations-lambda.proxy-format).

Configure `httpApi.payload` explicitly to ensure seamless migration.